XLForm/XL/Cell/XLFormSelectorCell.m

```diff
@@ -274,6 +283,8 @@ -(void)formDescriptorCellDidSelectedWithFormController:(XLFormViewController *)c
                                                                   }]];
             }
             [formViewController presentViewController:alertController animated:YES completion:nil];
+            // Fix for "Snapshotting a view that has not been rendered results in an empty snapshot." bug
+            [[alertController view] layoutIfNeeded];
```

Comment:
I have created a sample Xcode project to highlight this issue. This can be replicated consistently with Xcode 7.3.1 with iOS 9.3 Simulator (iPad, iPad Air 2, iPad Pro). Xcode console shows warnings every time the action sheet is presented. You can uncomment line 287 of XLFormSelectorCell.m to see how it stops the warnings. 

@mats-claassen Let me know if you prefer me to exclude this as this seems to be fixing a UIKit framework bug. Happy to hear what your thoughts are on this.

[TEST.zip](https://github.com/xmartlabs/XLForm/files/329269/TEST.zip)